kegtab/src/main/java/org/kegbot/backend/Backend.java

```diff
@@ -24,6 +24,7 @@
 import org.codehaus.jackson.JsonNode;
 import org.kegbot.app.util.TimeSeries;
 import org.kegbot.proto.Api.RecordTemperatureRequest;
+import org.kegbot.proto.Models;
```

Comment:
This import could be more specific since you are only using `Models.ThermoSensor`